### PR TITLE
Add documentation for containerized native builds on remote daemons

### PR DESCRIPTION
**Pull request by jonathan-meier**
Doc for PR #14635.

### CODE REVIEW

**Review by zakkak — CHANGES_REQUESTED** [review]
Thanks for the PR @jonathan-meier :)
Please see my comments for a couple of typos and a suggestion.

File: docs/src/main/asciidoc/building-native-image.adoc [hunk 1]
```diff
@@ -393,7 +393,14 @@ If you want to explicitely select the container runtime, you can do it with:
 
 These are normal Quarkus config properties, so if you always want to build in a container
 it is recommended you add these to your `application.properties` in order to avoid specifying them every time.
+====
+
+[TIP]
+====
+If your application JAR builds fine but you get an invalid path error for the JAR when trying to create a native executable using a container build, you're most likely using a remote daemon for your container runtime.
```
[the remark this answers is not in the document]
Comment (jonathan-meier) [reply docs/src/main/asciidoc/building-native-image.adoc]: Good suggestion, I reworded this part a bit to fluently include a copy of the error.

**Comment by jonathan-meier** [comment]
Thanks for the feedback! I fixed the typos and addressed your suggestion.

**Review by zakkak — APPROVED** [review]
Thanks for the fixes!

It looks good to me.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**